moped-editor/src/utils/mapHelpers.js

```diff
@@ -114,10 +101,10 @@ export const mapStyles = {
 /**
  * Common styles of map components in ProjectComponentsMap and ProjectComponentsMapView
  */
-export const makeCommonComponentsMapStyles = theme => ({
+export const makeCommonComponentsMapStyles = (theme) => ({
   speedDial: {
-    right: "3.5rem !important",
-    bottom: "1.7rem !important",
+    right: "49px !important",
```

Comment:
Most of the pixel numbers came from preventing the geocoder replacement component from overlapping other map controls (they use 10px for spacing for their controls and specific widths for button, etc.). This could be tightened up, but that could be a backlog issue.